src/libsync/theme.cpp

```diff
@@ -688,7 +688,7 @@ bool Theme::wizardEnableWebfinger() const
     return false;
 }
 
-QVector<QPair<QString, QUrl>> Theme::urlButtons() const
+QVector<std::tuple<QString, QString, QUrl>> Theme::urlButtons() const
```

Comment:
How about adding an actual class to house the data? This should make the code easier to read and understand.

Reply:
Its use is currently very limited, adding yet another class is currently probably not worth it.

Reply:
Hmm I could directly return an icon

Reply:
Ah I can't as this code is generated ....

Reply:
Good enough for me.